tests/test_stimuli.py

```diff
@@ -247,5 +250,70 @@ def test_fields_present3(self, identifier, field):
         stimulus_set = brainscore.get_assembly(f"brendel.Geirhos2021_{identifier}")
         assert hasattr(stimulus_set, field)
 
+@pytest.mark.private_access
+class TestBaker2022:
+
+    # general tests
+    @pytest.mark.parametrize('identifier', [
+        'normal',
+        'inverted'
+    ])
+    def test_stimulus_set_exist(self, identifier):
+        full_name = f'Baker2022_{identifier}_distortion'
+        stimulus_set = brainio.get_stimulus_set(full_name)
+        assert stimulus_set is not None
+        assert stimulus_set.identifier == full_name
+
+    # tests number of images
+    @pytest.mark.parametrize('identifier, num_images', [
+        ('normal', 716),
+        ('inverted', 360),
+    ])
+    def test_num_stimuli(self, identifier, num_images):
+        stimulus_set = brainio.get_stimulus_set(f'Baker2022_{identifier}_distortion')
+        assert len(stimulus_set) == num_images
+        assert len(np.unique(stimulus_set["stimulus_id"])) == num_images
+
+    # tests stimulus_set coords. Ensure normal/inverted only have their respective stimuli
+    @pytest.mark.parametrize('field', [
+        'stimulus_id',
+        'animal',
+        'image_type',
+        'image_number',
+        "orientation",
+    ])
+    @pytest.mark.parametrize('identifier', [
+        'normal',
+        'inverted',
+    ])
+    def test_fields_present(self, identifier, field):
+        stimulus_set = brainscore.get_stimulus_set(f'Baker2022_{identifier}_distortion')
+        assert hasattr(stimulus_set, field)
+
+    # make sure there are at least whole, frankenstein in each stimulus_set. Inverted does not have fragmented stimuli.
+    @pytest.mark.parametrize('identifier, count', [
+        ('normal', 3),
+        ('inverted', 2),
+    ])
+    def test_distortion_counts(self, identifier, count):
+        stimulus_set = brainscore.get_stimulus_set(f'Baker2022_{identifier}_distortion')
+        assert len(np.unique(stimulus_set["image_type"])) == count
+        assert "w", "f" in set(stimulus_set["image_type"])
```

Comment:
```suggestion
        assert "w" in set(stimulus_set["image_type"])
        assert "f" in set(stimulus_set["image_type"])
```
otherwise you run `assert "w"` (the rest is error message)